TCMSApp/src/client/app/tests/test-create.html

```diff
@@ -188,14 +189,14 @@
               <button type="button" class="btn btn-danger"><i class="fa fa-times"></i> Cancel</button>
             </div>
 
-          </div> <!-- closing panel-body -->
+          <!-- </div> closing panel-body -->
```

Comment:
As I can see, we should reconsider our UI with whole crew.
Therefore, I decided to leave some trace for return till Monday :)